### DISCUSSION TITLE
Container is running on IPs ranges that aren't in my network

### DISCUSSION THREAD

**Discussion by anthonyseto** [mrlt8/docker-wyze-bridge#1217] [Q&A]
I just upgraded to v2.9.2 and now when I start the container, the 3rd octet of the IP is changes with every start which doesn't align to an acceptable IP range on my network. 

My IP ranges should be 192.168.0.X. Instead, when I start the container, I get 192.168.Y.X where both X and Y are changing. WTF am I missing here that is causing this?

I'm using an RPi that is 192.168.0.3 so this should be starting running on 192.168.0.3:5000 not a random address like 192.168.112.2 etc.

**Comment by mrlt8** [comment]
Is this inside the docker container? I believe docker sets its own subnet within the container and this isn't related to your network:

<img width="511" alt="docker" src="https://github.com/mrlt8/docker-wyze-bridge/assets/67088095/4e637399-9d80-4473-b575-a6feb4b6b2f2">

**Reply by anthonyseto** [reply]
Yep. I had another network issue that I resolved that fixed access to the container. I was chasing the wrong issue because when I started the container, the output would say "running on https://localhost:5000" followed by "running on https://192.168.X.X:5000" and I was thinking that it was showing me the IP on my network. False alarm. Thanks for the continued work on this project!
